code/game/gamemodes/nuclear/nuclear_challenge.dm

```diff
@@ -39,8 +40,10 @@
 	U.hidden_uplink.uplink_owner= "[user.key]"
 	U.hidden_uplink.uses = 280
 	U.hidden_uplink.mode_override = /datum/game_mode/nuclear //Maybe we can have a special set of items for the challenge uplink eventually
+	config.shuttle_refuel_delay = CHALLENGE_SHUTTLE_DELAY
```

Comment:
in the off chance a high RP server has a longer refuel delay than the shuttle delay it probably shouldn't lower it.